sources/dataset.py

```diff
@@ -186,11 +186,11 @@ def group_image_label(self, image):
                 a[mask] = root_id
         return Image.fromarray(a, mode=image.mode)
 
-    def _preprocess(self, datadir, image_filename, aggregate, labelling=True):
+    def _preprocess(self, image_filename, output_dir, aggregate, labelling=True):
         """Resize/crop then save the training & label images
 
-        :param datadir: str
         :param image_filaname: str
+        :param output_dir: str
```

Comment:
Is is possible to update the docstring format? With numpydoc style?

Reply:
I suggest to open another issue to address this.

Reply:
ok